### PR TITLE
Update 2024-cloud-native-sustainability-week.md

### PR DESCRIPTION
**Pull request by humblec**
Add the location for Bangalore Sustainability week meet-up.

### CODE REVIEW

**Comment by vallss** [comment]
Hi @humblec, do you also know when the meetup will take place? If so, please add the date too 😊

**Comment by humblec** [comment]
> Hi @humblec, do you also know when the meetup will take place? If so, please add the date too 😊

@vallss updated the same. ptal. thanks!

File: website/content/en/events/2024-cloud-native-sustainability-week.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ After the successful [Sustainability Week 2023](https://tag-env-sustainability.c
 | 5 | Tokyo, Japan | Oct. 4 | [Link](https://community.cncf.io/events/details/cncf-cloud-native-community-japan-presents-cncf-cloud-native-sustainability-week-2024-local-meetup-tokyo/) | Sunyanan Choochotkeaw ([Cloud Native Community Japan](https://community.cncf.io/cloud-native-community-japan/))
 | 6 | Milan, Italy | Sep. 30 & Oct. 7 | [Link Sep. 30](https://www.meetup.com/digital-platform-builders/events/303350215) - [Link Oct. 7](https://www.meetup.com/digital-platform-builders/events/303350378) |  Michel Murabito, Valeria Salis, Ludovica Bonaldo
 | 7 | Sao Paulo, Brazil | Oct. 22 | [Link](https://community.cncf.io/events/details/cncf-cloud-native-sao-paulo-presents-27-cloud-native-sao-paulo-sustainability-day-na-aws/) | Carol Valencia, Tiago Reichert, Pedro Ignácio, Erivaldo Lopes
-| 8 | India | TBD | TBD | Saiyam
+| 8 | India | Bangalore| Oct. 19 | Prakash Mishra, Humble Devassy Chirammal, Kiran Mova
```
Comment (guidemetothemoon) [inline website/content/en/events/2024-cloud-native-sustainability-week.md:38]:
```suggestion
| 8 | India, Bangalore | Oct. 19 | TBD | Prakash Mishra, Humble Devassy Chirammal, Kiran Mova
```

**Review by guidemetothemoon — APPROVED** [review]
LGTM, thanks! 
P.S. FYI, I added a minor change directly to the PR to align the column values according to the rest of entries😊

**Comment by vallss** [comment]
Thanks @humblec!  And thanks @guidemetothemoon for the changes and merge!

**Pull request merged**